### DISCUSSION TITLE
NOVAthesis Word 1.1.0

### DISCUSSION THREAD

**Discussion by joaomlourenco** [joaomlourenco/novathesis_word#5] [Q&A] [answered]
Now there is a single cove for PhD and another for MSc (with subtitle).  Remove the subtitle if it does not apply to your document.

<hr /><em>This discussion was created from the release <a href='https://github.com/joaomlourenco/novathesis_word/releases/tag/1.1.0'>NOVAthesis Word 1.1.0</a>.</em>

**Comment by nesipac** [comment]
Hello! The page numeration for the first part (roman numeration) is not correct and I cannot change it, can you help please?

**Reply by joaomlourenco** [reply]
The pagination for the Word version of the templates considers double-sided pages and, thus, there are "blank pages" that will only show up in the PDF.  
I believe that, if we consider the blank pages in the PDF, the page numbering is all right.
I'll add a ticket/issue to design a new version of the template(s) with continuous pages.

**Reply by joaomlourenco** [reply]
Não percebi o problema. 
O problema que fala está no template original? Ou no seu documento?
Se está no template original, explique claramente o problema. Se está no seu documento apenas, não vejo como ajudar.

**Comment by AnaRebocho** [comment]
Hello! 
 I can't download the template in word version. Can you help me?

**Reply by joaomlourenco** [reply]
Why not? What is the problem?

**Reply by AnaRebocho** [reply]
I can't find a place to download.
When selecting the " novathesis_word/novathesis_word-msc-EN-v1.3.dotx " does not download

**Reply by joaomlourenco** [reply, marked as answer]
And what do you see when you click?  A a link in blue, saying "View raw"?  Did you try to follow the link?

**Comment by pallomasilva2012** [comment]


**Reply by joaomlourenco** [reply]
Não vejo quaisquer anexos na sua mensagem aqui no GitHub.

O que quer dizer com um problem na sequência das páginas em romano?  Se exportar o ficheiro para PDF, verá que aparecem páginas em branco a seguir à capa e folha de rosto e assim. A página "v" é o copyright.

Quanto aos títulos, não percebo qual é o problema. Não anexou imagens nenhumas e não o descreveu adequadamente.

**Comment by pallomasilva2012** [comment]


**Reply by joaomlourenco** [reply]
@pallomasilva2012 
Acabei de resolver o problema da numeração das páginas.
A versão que está no repositório já está corrigida.

**Comment by pallomasilva2012** [comment]
Olá  Professor João,  boa tarde. Que maravilha. Sera que tem como eu
corrigir a versão que já tenho a tese escrita?


A segunda, 16/03/2026, 17:22, João M. Lourenço ***@***.***>
escreveu:

**Reply by joaomlourenco** [reply]
Sim. Se me enviar o ficheiro DOCX por email, eu faço isso.

**Comment by pallomasilva2012** [comment]


**Reply by joaomlourenco** [reply]
@pallomasilva2012 
Envie-me um email diretamente (para o meu email) com a tese. Obrigado.

**Comment by pallomasilva2012** [comment]
Olá  Professor João,  bom dia.

Eu ainda estou a dar voltas e agora pediram me para atualizar o "logo" da
faculdade nas duas capas da tese.
Já tentei fazer de tudo, mas quando tento alterar desconfigura tudo.
Como poderia fazer para substituir o logo.
Obrigada

**Reply by joaomlourenco** [reply]
Fazer download da versão nova com o logo correto, selecionar as capas, copiar, colar na tese, selecionar as capas antigas, apagar. 

Ou então fazer ao contrário e copiar os capítulos escritos para o novo documento/template. 

Se não, é pedir ajuda a um colega que tenha experiência com o Word ou então aos funcionários da Biblioteca.

**Comment by pallomasilva2012** [comment]
Tá bem, vou tentar fazer isso. Muito Obrigada Professor.

A quarta, 8/04/2026, 16:09, João M. Lourenço ***@***.***>
escreveu:

**Comment by pallomasilva2012** [comment]
Não consegui Professor.  Não sei o que fazer : (

A quarta, 8/04/2026, 21:50, Palloma Silva ***@***.***>
escreveu:
